### PR TITLE
19/cookies modal/feat

### PR DESCRIPTION
**Pull request by ccoutts2**
- Basic cookie functionality accept or reject request
- Basic styling for time being

### CODE REVIEW

File: src/components/buttons/Button/Button.tsx
Comment (kabirseth4) [inline src/components/buttons/Button/Button.tsx:1]: Not necessarily essential, but I think we should start extending our props from the HTML elements where possible:

```typescript
interface ButtonProps extends React.ButtonHTMLAttributes<HTMLButtonElement> {
  label: string;
}

export const Button = ({ label, ...defaultProps }: ButtonProps) => {
  return (
    <button
      className="mx-0 my-[0.75rem] h-auto w-auto bg-[#744442] px-[1.8rem] py-2 text-xs uppercase tracking-[1px] text-[#f3eee8]"
      {...defaultProps}
    >
      {label}
    </button>
  );
};
```

File: src/components/Cookie/Cookie.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
+"use client";
+
+import { useState, useEffect, useRef } from "react";
+import { Button } from "@/components";
+import Cookies from "js-cookie";
+import Link from "next/link";
+
+const Cookie = () => {
+  const [showCookie, setShowCookie] = useState(false);
+  const [consent, setConsent] = useState(false);
```
Comment (kabirseth4) [inline src/components/Cookie/Cookie.tsx:10]: Will probs need to chuck this in a context at some point, but fine here for now I think

**Review by kabirseth4 — CHANGES_REQUESTED** [review]

**Review by kabirseth4 — CHANGES_REQUESTED** [review]
Small change with button props

File: src/components/buttons/Button/Button.tsx
Comment (ccoutts2) [reply src/components/buttons/Button/Button.tsx:1]: I agree. I've just started doing this elsewhere coincidentally

**Pull request merged**